docs/reference/troubleshooting/common-issues/mapping-explosion.asciidoc

```diff
@@ -41,13 +41,16 @@ timing out in the browser's Developer Tools Network tab.
 doesn't normally cause problems unless it's combined with overriding
 <<mapping-settings-limit,`index.mapping.total_fields.limit`>>. The
 default `1000` limit is considered generous, though overriding to `10000`
-doesn't cause noticable impact depending on use case. However, to give
+doesn't cause noticeable impact depending on use case. However, to give
 a bad example, overriding to `100000` and this limit being hit
 by mapping totals would usually have strong performance implications.
 
 If your index mapped fields expect to contain a large, arbitrary set of
 keys, you may instead consider:
 
+* Setting <<mapping-settings-limit,`index.mapping.total_fields.ignore_dynamic_beyond_limit`>> to `true`.
+Instead of rejecting documents that exceed the field limit, this will avoid adding dynamic fields once the limit is reached.
```

Comment:
> can you be explicit that documents with additional fields won't be rejected anymore.

Isn't "Instead of rejecting documents that exceed the field limit, [...]" already explicit enough? Could you suggest a different wording?